spring-integration-hazelcast/build.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: 'java'
+apply plugin: 'maven'
+
+group = 'org.springframework.integration'
+
+description = 'Spring Integration Hazelcast Support'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+
+
+repositories {
+        
+     maven { url "http://repo.maven.apache.org/maven2" }
+}
+
+dependencies {
+    compile group: 'org.springframework', name: 'spring-context', version:'4.1.5.RELEASE'
```

Comment:
Hi, @erenavsarogullari !

I have some time today, so let me to provide some comments even on the weekend :smile: 

No need in this direct dependency: it is transitive from the `SI-core`